Makefile

```diff
@@ -349,6 +350,36 @@ build-deb: build-release
 				"/build/target/${TARGET}/release/logdna-agent=/usr/bin/logdna-agent" \
 				"packaging/linux/logdna-agent.service=/lib/systemd/system/logdna-agent.service"'
 
+RPM_VERSION=1
+RPM_ARCH_NAME_x86_64=amd64
+RPM_ARCH_NAME_aarch64=arm64
```

Comment:
I believe RPM uses x86_64 and aarch64 for it's packages, so we don't need to lookup here

Reply:
Good to know. Addressed in commit https://github.com/logdna/logdna-agent-v2/pull/251/commits/ceb1134e57dba99b72072e4c35c339fd78e343b8.